src/app/form/(formSteps)/FormLayout.tsx

```diff
@@ -1,9 +1,66 @@
+import { allSections, getCurrentFormProgress } from "@form/_utils/formProgress";
+import { RequiredMarker } from "@trussworks/react-uswds";
+
+type CompletionState = "complete" | "current" | "incomplete";
+
 // Separated this into a separate testable component because as of writing, Jest does not support testing NextJs asynchronous server components (https://nextjs.org/docs/app/guides/testing/jest)
 export const FormLayout = (props: { children?: React.ReactNode; pathname: string }) => {
+  const { section: currentSection } = getCurrentFormProgress(props.pathname);
+  const currentSectionIndex = allSections.findIndex(
+    (sections) => sections.id === currentSection.id,
+  );
+
   return (
-    <div>
-      <h1 className="margin-top-4">{props.pathname}</h1>
-      <div className="margin-top-4">{props.children}</div>
-    </div>
+    <>
+      <div className="usa-step-indicator" aria-label="progress">
+        <div className="usa-step-indicator__segments">
+          {allSections.map((sections, sectionIndex) => {
+            let completionState: CompletionState;
+            switch (true) {
+              case sectionIndex < currentSectionIndex:
+                completionState = "complete";
+                break;
+              case sectionIndex === currentSectionIndex:
+                completionState = "current";
+                break;
+              case sectionIndex > currentSectionIndex:
+                completionState = "incomplete";
+                break;
+              default:
+                throw new Error(`Unexpected logic path: ${sectionIndex}, ${currentSectionIndex}`);
+                break;
+            }
+
+            const liSegmentClassSuffix = {
+              complete: "complete",
+              current: "current",
+              incomplete: null,
+            }[completionState];
+
```

Comment:
[boulder] I would add some sort of context to the stepper to describe to screen reader users where they are in the progress of the form:

```suggestion

            const stepNumber = sectionIndex + 1;
            const totalSteps = allSections.length;
            
            let ariaLabel = `Step ${stepNumber} of ${totalSteps}: ${sections.progressBarTitle}`;
            if (completionState === "complete") {
              ariaLabel += " (completed)";
            } else if (completionState === "current") {
              ariaLabel += " (current step)";
            }
```